documentation/en/source/pages/command-line/migration.rst

```diff
@@ -18,4 +18,14 @@ Sub Commands
 ``list`` sub-command
 ~~~~~~~~~~~~~~~~~~~~
 
-Displays all the migrations for the user.
\ No newline at end of file
+Displays all the migrations for the user.
+
+
+``delete`` sub-command
+~~~~~~~~~~~~~~~~~~~~~~
+
+Deletes a complete migration. The options are:
```

Comment:
From my opinion, "complete migration" can be understood as "finished migration" or "full migration". Can be a little bit confusing.